### DISCUSSION TITLE
Version v1.2.0 Released!

### DISCUSSION THREAD

**Discussion by hollowaykeanho** [ZORALab/Hestia#382] [Announcements]
We're proud to announce we had release version `v.1.2.0` of ZORALab's Hestia. The fixes primarily focuses on supporting Hugo theme's performance from limited 50 pages to 100 pages and re-organize all components into a inter-operable standard libraries.

This enables us to work on Phase 2 with documentation supports.

Also, starting from this version, we're supporting simplified script Global Mandarin （华语-国际） `zh-hans` language.

For more information visit: https://github.com/ZORALab/Hestia/releases/tag/v1.2.0

**Comment by JeanShuralyov** [comment]
Congrats! Finally after 2 months of caved-in development. XD
